src/core/state.go

```diff
@@ -988,6 +990,12 @@ func (state *BuildState) ForConfig(config ...string) *BuildState {
 	return s
 }
 
+func (state *BuildState) ForSubrepo(name string) *BuildState {
+	s := state.ForConfig()
```

Comment:
is ForConfig still relevant or is it obsoleted by ForSubrepo?

Reply:
I'm just using it here to clone the state object. I suppose I could move that out into something more explicit.